cpp/CMakeLists.txt

```diff
@@ -129,8 +134,13 @@ endif()
 
 set(CUDF_CXX_FLAGS "")
 set(CUDF_CUDA_FLAGS "")
-set(CUDF_CXX_DEFINITIONS "")
-set(CUDF_CUDA_DEFINITIONS "")
+if(CUDF_ENABLE_NVTX_DEBUG_RANGE)
+  set(CUDF_CXX_DEFINITIONS "CUDF_ENABLE_NVTX_DEBUG_RANGE")
```

Comment:
Do we want to append to `CUDF_CXX/CUDA_DEFINITIONS` here instead to avoid resetting the variable?